scene/gui/color_picker.cpp

```diff
@@ -222,6 +222,23 @@ void ColorPicker::add_preset(const Color &p_color) {
 		bt_add_preset->hide();
 }
 
+void ColorPicker::erase_preset(const Color &p_color) {
+
+	if (presets.find(p_color)) {
+		presets.erase(presets.find(p_color));
+		preset->update();
+	}
+}
+
+Array ColorPicker::get_presets() const {
```

Comment:
Shouldn't this return a `PoolColorArray`?
Not sure if there's an advantage but all the methods I looked at that return single type arrays use `Pool*Array`s if a matching one exists.

Reply:
yes, it seems to be a better option. I'm changing it.